internal/impl/laravel/5.8/handler/generator/model_generator.go

```diff
@@ -0,0 +1,146 @@
+package generator
+
+import (
+	"asher/internal/api"
+	"asher/internal/api/codebuilder/php/builder"
+	"asher/internal/api/codebuilder/php/builder/interfaces"
+	"asher/internal/api/codebuilder/php/core"
+	"github.com/iancoleman/strcase"
+)
+
+type ModelGenerator struct {
+	classBuilder interfaces.Class
+	fillables []string
+	hidden []string
+	timestamps bool
+	createValidationRules map[string]string
+	updateValidationRules map[string]string
+}
+
+/**
+ Creates a new instance of this generator with a new interfaces.Class
+ */
+func NewModelGenerator() *ModelGenerator {
+	return &ModelGenerator{
+		classBuilder: builder.NewClassBuilder(),
+	}
```

Comment:
set all members to their defaults... dont add nil in here... this way you avoid unecessary if statements like in the method below